sources/node/KalturaClientBase.js

```diff
@@ -674,3 +681,12 @@ KalturaConfiguration.prototype.setLogger = function(logger){
 KalturaConfiguration.prototype.getLogger = function(){
 	return this.logger;
 };
+
+/**
+ * @param host - hostname or ip address of proxy
+ * @param port - port of proxy
+ */
+KalturaConfiguration.prototype.setProxy = function(host, port)
```

Comment:
It is optionally used by the application creating the KalturaClient to set proxy values of the host, like below:

`var config = new Kaltura.kc.KalturaConfiguration(partnerID);
config.setProxy('localhost', 3128);
var kalturaClient = new Kaltura.kc.KalturaClient(config);

`